src/go/rpk/pkg/cli/cmd/wasm/deploy_test.go

```diff
@@ -0,0 +1,118 @@
+package wasm
+
+import (
+	"bytes"
+	"fmt"
+	"testing"
+
+	"github.com/Shopify/sarama"
+	"github.com/Shopify/sarama/mocks"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/afero"
+	"github.com/stretchr/testify/require"
+)
+
+type FileInformationTest struct {
+	name	string
+	content	string
+}
+
+func TestNewDeployCommand(t *testing.T) {
+	tests := []struct {
+		name		string
+		producer	func(bool, int32) (sarama.SyncProducer, error)
+		fileInformation	FileInformationTest
```

Comment:
Since these tablel tests are super nested, it's better to name things as short as possible. In this case, just `fileInfo` would be OK (same for the struct name: `fileInfo`).

Reply:
Done